andela/matterhorn-cfh
# #151487637 Ensure Travis build successfully

### What does this PR do?
Seek to make travis build pass/build successfully

### Description of Task to be completed?
Run tests and display code quality badges based on test results.

### How should this be manually tested?
- For test cd to project directory `npm install` t
- Then do `npm run coverage` to see local generated results
- To see badge on repository, visit travis build page during pull request build
and ensure all tests passed, results would be sent to coveralls
- Click badge on the repo to confirm  

### Any background context you want to provide?
1. Ensure all packages required by package.json are installed;

### What are the relevant pivotal tracker stories?
#151487637 Ensure Travis build successfully

### Screenshots (if appropriate)
<img width="1376" alt="fix-travis-build" src="https://user-images.githubusercontent.com/20375577/30938760-11effbd4-a3d3-11e7-99c7-b489343bd205.png">

### Questions:
N/A
